superset/assets/javascripts/explorev2/stores/store.js

```diff
@@ -407,6 +407,21 @@ export const visTypes = {
     ],
   },
 
+  bullet: {
+    label: 'Bullet Chart',
```

Comment:
the `bullet` object needs to contain `requiresTime: false,` so the time controls are not rendered.